staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -330,6 +330,12 @@ const (
 	// the 'nominalConcurrencyShares' field of the 'limited' section of a
 	// priority level.
 	ZeroLimitedNominalConcurrencyShares featuregate.Feature = "ZeroLimitedNominalConcurrencyShares"
+
+	// owner: @chenchun
+	// alpha: v1.32
+	//
```

Comment:
shouldn't this have a KEP?

Reply:
Yes, this requires a KEP.

cc @deads2k

Reply:
OK, I'll try to write a KEP

Reply:
It should - I will wait with reviewing until this has a KEP.

Reply:
I've written a KEP, PTAL https://github.com/kubernetes/enhancements/pull/4934